diag_manager/fms_diag_time_utils.F90

```diff
@@ -27,17 +27,18 @@
 module fms_diag_time_utils_mod
 
 use time_manager_mod, only: time_type, increment_date, increment_time, get_calendar_type, NO_CALENDAR, leap_year, &
-                            get_date, get_time
+                            get_date, get_time,  operator(>), operator(<), operator(-)
 use diag_data_mod,    only: END_OF_RUN, EVERY_TIME, DIAG_SECONDS, DIAG_MINUTES, DIAG_HOURS, DIAG_DAYS, DIAG_MONTHS, &
                             DIAG_YEARS
 USE constants_mod,    ONLY: SECONDS_PER_DAY, SECONDS_PER_HOUR, SECONDS_PER_MINUTE
-use fms_mod,          only: fms_error_handler
+use fms_mod,          only: fms_error_handler, error_mesg, FATAL
```

Comment:
I understand why, but this is inconsistent with the other modules that use mpp_error